src/stacked_borrows.rs

```diff
@@ -286,7 +286,7 @@ impl<'tcx> Stack {
 
     /// Test if a memory `access` using pointer tagged `tag` is granted.
     /// If yes, return the index of the item that granted it.
-    fn access(
+    pub(crate) fn access(
```

Comment:
These should be private again after you moved the methods on `Stacks` back into this module.